src/SerilogTraceListener/SerilogTraceListener.cs

```diff
@@ -79,7 +81,7 @@ public override bool IsThreadSafe
 
         public override void Close()
         {
-            logger = new SilentLogger();
+            this.logger = Log.Logger.ForContext("SourceContext", context);
```

Comment:
Yes, you could just not override, i.e. have the default do nothing, as the concept doesn't really translate across into Serilog.

If you look at TextWriterTraceListener, on Close() it closes the underlying file and sets it to null, but the next call to Write() just opens up the file again, i.e. the name "close" and the description in docs is a bit misleading.

I will just remove it.